src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,39 @@
+import "./NavBar.scss";
+import Menu from "../../assets/images/Menu.svg";
+import Search from "../../assets/images/Search.svg";
+import Upload from "../../assets/images/Upload.svg";
+import Notifications from "../../assets/images/Notifications.svg";
+import useWindowSize from "../../hooks/useWindowsSize.js";
+
+const NavBar = () => {
+  const windowIsDesktop = useWindowSize(1024);
+
+  const mobileNav = (
+    <div className="NavBar">
+      <h2 className="NavBar__title">ZINESTORE</h2>
```

Comment:
can we set the zinestore string as a variable so when we come up with a new name we only have to change it in one place rather than in multiple?